freqtrade/optimize/backtesting.py

```diff
@@ -204,12 +204,12 @@ def load_bt_data(self) -> Tuple[Dict[str, DataFrame], TimeRange]:
         """
         self.progress.init_step(BacktestState.DATALOAD, 1)
 
-        if self.config['freqaimodel']:
-            self.required_startup += int((self.config['freqai']['train_period']*86400) /
-                                    timeframe_to_seconds(self.config['timeframe']))
+        if self.config['freqai']['train_period'] > 0:
```

Comment:
You should use the following here to allow this to pass (not just unittests) with any config that is not using freqAI (does not have that configured).

```suggestion
        if self.config.get('freqai', {}).get('train_period') > 0:
```

Using this "optional" style will not be required throughout i guess - but for interfacing parts - always assume people have an existing configuration - and that behavior should not break.

Reply:
Thanks, changed it.